py/client2/README.md

```diff
@@ -16,60 +38,40 @@ pip3 install numpy
 pip3 install cython
 ```
 
-Set the `CMAKE_PREFIX_PATH` environment variable to include all DH's cpp-client dependencies.
-Your `CMAKE_PREFIX_PATH` should look something like:
+## Building the Deephaven shared library for Python
+
+First, enter the Python client directory:
 
 ```
-CMAKE_PREFIX_PATH=\
-/home/cfs/dhcpp/local/abseil:\
-/home/cfs/dhcpp/local/arrow:\
-/home/cfs/dhcpp/local/boost:\
-/home/cfs/dhcpp/local/cares:\
-/home/cfs/dhcpp/local/flatbuffers:\
-/home/cfs/dhcpp/local/gflags:\
-/home/cfs/dhcpp/local/grpc:\
-/home/cfs/dhcpp/local/immer:\
-/home/cfs/dhcpp/local/protobuf:\
-/home/cfs/dhcpp/local/re2:\
-/home/cfs/dhcpp/local/zlib
+cd ${DHROOT}/py/client2
 ```
 
-(if you used `build-dependencies.sh` to build them, the script created a `env.sh` script for you
-that you can source to set CMAKE_PREFIX_PATH to the targets it created for you).
-
-Then, in the `py/client2` directory, run:
+Then run these commands to build the Deephaven shared library:
 
 ```
-  $ python3 setup.py build_ext --inplace
+rm -rf build *.so && \
+CFLAGS="-I${DHLIB}/deephaven/include" LDFLAGS="-L${DHLIB}/deephaven/lib -L${DHLIB}/protobuf/lib -L${DHLIB}/arrow/lib -L${DHLIB}/grpc/lib -L${DHLIB}/abseil/lib -L${DHLIB}/re2/lib" python setup.py build_ext -i
 ```
 
-Other environment variables useful for debugging:
+Once built, a shared object with the binary python module should show up, named like
+`pydeephaven.cpython-38-x86_64-linux-gnu.so`.
 
-* `VERBOSE=1` prints detailed compiler invocations, including flags like `-I`.
-* `PARALLEL=9` run up to 9 parallel processes, useful to speed up the compilation.  Set the value to your number of CPUS + 1.
```

Comment:
You can get the number of CPUs on a linux system with `getconf _NPROCESSORS_ONLN`.  Do we want to use that?